src/naming_screen.c

```diff
@@ -2177,6 +2189,18 @@ static const struct NamingScreenTemplate sWaldaWordsScreenTemplate =
     .title = gText_TellHimTheWords,
 };
 
+static const u8 sText_DexRiddle[] = _("   Which species?");
```

Comment:
also probably not gonna do this one for similarish reasons to above and also was following the pattern Doodle did with sText_RivalsName. also @purrfectdoodle in case you want to change it when it comes time to implement the actual script tho